Sources/Swift/Integrations/SessionReplay/SentryReplayOptions.swift

```diff
@@ -153,60 +175,100 @@ public class SentryReplayOptions: NSObject, SentryRedactOptions {
      *               view renderer, please report the issue on [GitHub](https://github.com/getsentry/sentry-cocoa). Eventually, we will
      *               mark this feature as stable and remove the experimental flag, but will keep it disabled by default.
      */
-    public var enableFastViewRendering = false
+    public var enableFastViewRendering: Bool
 
     /**
      * Defines the quality of the session replay.
      * Higher bit rates better quality, but also bigger files to transfer.
      */
-    var replayBitRate: Int {
-        quality.rawValue * 20_000 + 20_000
+    internal var replayBitRate: Int {
```

Comment:
Why add all these? Isn't `internal` the default?

Reply:
My intention was to be explicit what's public facing and what not, so that any future maintainer visiting this property will reconsider if making it public is necessary. If no visibility is defined, some might be careless and think something like: "Oh we missed adding a `public` here, that's why it's not available".

Reply:
As internal is the default, it's a bit redundant, but I don't care too much about that.

Reply:
Removed the internal